File: src/multiquines.clj
(let[S"DATA"L(vec(.split S"\t"))I(.indexOf L(first *command-line-args*))](print(str(get L(+ I 1))(clojure.string/escape S{\tab"\\t"\newline"\\n"\""\\\""\\"\\\\"})(get L(+ I 2)))))

File: src/multiquines.fs
let S="DATA"
let L=S.Split '\t'
let I=Array.findIndex((=)(System.Environment.GetCommandLineArgs()).[1])L
printf"%s"(L.[I+1]+S.Replace("\\",@"\\").Replace("\t",@"\t").Replace("\n",@"\n").Replace("\"",@"\""")+L.[I+2])
